contracts/SpokePool.sol

```diff
@@ -209,8 +249,20 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
      **************************************/
 
     /**
-     * @notice Called by user to bridge funds from origin to destination chain.
-     * @dev The caller must first approve this contract to spend `amount` of `originToken`.
+     * @notice Called by user to bridge funds from origin to destination chain. Depositor will effectively lock
+     * tokens in this contract and receive a destination token on the destination chain. The origin => destination
+     * token mapping is stored on the L1 HubPool.
+     * @notice The caller must first approve this contract to spend `amount` of `originToken`.
+     * @notice The originToken => destinationChainId must be enabled.
+     * @notice This method is `payable` because the caller is able to deposit ETH if the originToken is WETH and this
+     * function will handle wrapping ETH.
+     * @param recipient Address to receive funds at on destination chain.
+     * @param originToken Token to lock into this contract to initiate deposit.
+     * @param amount Amount of tokens to deposit. Will be amount of tokens to receive less fees.
+     * @param destinationChainId Denotes network where user will receive funds from SpokePool by a relayer.
+     * @param relayerFeePct % of deposit amount taken out to incentivize a fast relayer.
+     * @param quoteTimestamp Timestamp used by relayers to compute this deposit's "realized LP fee %" which is paid
+     * to LP pool on HubPool.
```

Comment:
```suggestion
     * @notice The caller must first approve this contract to spend amount of originToken.
     * @notice The originToken => destinationChainId must be enabled.
     * @notice This method is payable because the caller is able to deposit ETH if the originToken is WETH and this
     * function will handle wrapping ETH.
     * @param recipient Address to receive funds at on destination chain.
     * @param originToken Token to lock into this contract to initiate deposit.
     * @param amount Amount of tokens to deposit. Will be amount of tokens to receive less fees.
     * @param destinationChainId Denotes network where user will receive funds from SpokePool by a relayer.
     * @param relayerFeePct % of deposit amount taken out to incentivize a fast relayer.
     * @param quoteTimestamp Timestamp used by relayers to compute this deposit's realizedLPFeePct which is paid
     * to LP pool on HubPool.
```